ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/433__data-science-pipelines-general.robot

```diff
@@ -0,0 +1,80 @@
+*** Settings ***
+Documentation       Test suite for OpenShift Pipeline API
+
+Resource            ../../../Resources/RHOSi.resource
+Resource            ../../../Resources/ODS.robot
+Resource            ../../../Resources/Common.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataSciencePipelines.resource
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource            ../../../Resources/Page/Operators/OpenShiftPipelines.resource
+Library             DateTime
+Library             ../../../../libs/DataSciencePipelinesAPI.py
+Suite Setup         General Suite Setup
+Suite Teardown      General Suite Teardown
+
+
+*** Variables ***
+${PROJECT_USER2}=    dspa-project-user2
+${PROJECT_USER3}=    dspa-project-user3
+${S3_BUCKET}=    ods-ci-ds-pipelines
+
+
+*** Test Cases ***
+Verify Ods User Can Bind The Route Role
+    [Documentation]    Verify Ods User Can Bind The Route Role
+    [Tags]      Sanity
+    ...         Tier1
+    ...         ODS-2209
+    Create A Data Science Projects    ${TEST_USER_2.USERNAME}    ${TEST_USER_2.PASSWORD}    ${TEST_USER_2.AUTH_TYPE}    ${PROJECT_USER2}    # robocop: disable:line-too-long
```

Comment:
why disabling the robocop alert? You could ditrbute the arguments on new lines

Reply:
I prefer having it in one line

Reply:
We keep the line lenght limit to improve code readability, I suggest to do the same here

Reply:
I think it is a preference :D

Reply:
Hi Berto. Didn't get the point. What rule of the style guide? If the line is long we can ignore it adding `# robocop: disable:line-too-long`.

Reply:
Hi Berto. Didn't get the point. What rule of the style guide? If the line is long we can ignore it adding `# robocop: disable:line-too-long`.

Reply:
I'm referring to the section `Lengths`, it says the agreed rule we follow. Of course it is possible to disable the alert, but I'm just saying we as a team agreed on those code style rules and it would be good to stick on those or try changing them if there are reasons

Reply:
@diegolovison what Berto means is that, as a team, we have agreed on this [code style guide](https://docs.google.com/document/d/11ZJOPI1uq-0Wl6a2V8fkAv_TQhfzp9t_IjXAheaJxmQ/edit#heading=h.ld8pghdw594b) and we try to follow it when summing code to ods-ci.

Also, we discussed recently again about line length and the general consensus was to keep it as it is (120)

Reply:
I did the change. Now the test for me is super hard to understand what a single line is doing